### PR TITLE
Add RoboForm Latest

### PR DESCRIPTION
**Pull request by agross**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/roboform.rb [hunk 1]
```diff
@@ -1,20 +1,20 @@
 cask 'roboform' do
-  version :latest
-  sha256 :no_check
+  version '8.4.8'
+  sha256 '6792112bc8327b624259f574ab4e3a50cf00a376f74feb187c50f85b3c15595d'
```
Comment (agross) [inline Casks/roboform.rb]: The URL does not change between versions. This why I specified `:latest`. Why the change?

Comment (commitay) [reply Casks/roboform.rb]: The appcast allows the Cask to be versioned.

Comment (agross) [reply Casks/roboform.rb]: So the idea is someone changes the appcast page -> checkpoint does no longer match. Then what?

Comment (commitay) [reply Casks/roboform.rb]: If the `version` / `sha256` is still the same, nothing. If the `version` / `sha256` has changed, the Cask needs to be updated.

Comment (agross) [reply Casks/roboform.rb]: > If the version / sha256 has changed, the Cask needs to be updated.

OK, this is what needs to be done as new versions change the dmg.

Comment (agross) [reply Casks/roboform.rb]: What happens if nobody updates the cask, but the manufacturer updates the dmg. I guess users won't be able to install then. Is that preferred to no version/no SHA verification?

Comment (commitay) [reply Casks/roboform.rb]: Yes.

Comment (agross) [reply Casks/roboform.rb]: OK. Is there any way provided by homebrew-cask to be notified about such updates?

(I hope you don't mind me asking such dumb questions, I'm a first-time contributor.)

**Review by victorpopkov — APPROVED** [review]
